src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -429,11 +436,10 @@ public void onResponse(final Call<ResponseBody> call, final Response<ResponseBod
                 try {
                     if (response.isSuccessful()) {
                         BufferedSource source = response.body().source();
-                        while (true) {
-                            QueryResult result = InfluxDBImpl.this.adapter.fromJson(source);
-                            if (result != null) {
-                                consumer.accept(result);
-                            }
+                        QueryResult result = InfluxDBImpl.this.adapter.fromJson(source);
```

Comment:
Why is this implementation changed, seem unrelated to what you want to achieve.

Reply:
IntelliJ IDE showed an error because of an endless loop